canvas/templates/canvas/problem_set_snippet.html

```diff
@@ -25,6 +25,9 @@
             </td>
             <td>
                 <a href="{% url 'course:question_view' uqj.question.pk %}" class="btn btn-primary">Open</a>
+                {% if user.is_teacher %}
+                    <a href="{% url 'course:question_edit' uqj.question.pk %}" class="btn btn-primary">Edit</a>
```

Comment:
Can you please the class to btn-warning? So its consistent with our other edit buttons.